models/data_frames/lights_status_data_frame.py

```diff
@@ -2,23 +2,11 @@
 from typing import List
 
 
-# TODO reimplement this class when the new data frames arrives
 class LightsDataFrame(DataFrame):
-    NUMBER_OF_LIGHTS = 8
 
-    def __init__(self, frame_id: int, frame_value: bytes):
+    def __init__(self, frame_id: int, frame_value: bytes, lights_status: List[bool]):
```

Comment:
What is the Boolean list represent !? The logic must not be in the mind of the developer specially that this the database model